src/main/java/fr/techad/edc/httpd/deploy/GetAndDeployService.java

```diff
@@ -106,13 +108,27 @@ private void synchronize(File directory, String docPath, boolean override) throw
     for (File f : filestoCopy) {
       String path1 = f.getPath();
       path1 = path1.replace(directory.getCanonicalPath(), "");
+      path1 = path1.replace("\\", "/");
+      String [] tmp=path1.split("/");
+       name = tmp[1];
+       if(!(name.equals("multi-doc.json")||name.equals("i18n"))) {
+         tmp[1]=name+"-new";
```

Comment:
Define **-new** and **-old** as constant in this classes (SUFFIX_NEW and SUFFIX_OLD). You minmiize the risk of errors and the factory is easier.